packages/create-starter/src/index.ts

```diff
@@ -44,35 +47,39 @@ export async function main() {
       name: 'kit',
       message: 'Which starter kit would you like to use?',
       choices: starters,
-      suggest: (input, choices) => Promise.resolve(choices.filter(c => c.title.includes(input))),
+      suggest: (input, choices) => Promise.resolve(choices.filter((c) => c.title.includes(input))),
     },
     {
       type: 'text',
       name: 'name',
       message: 'What is the name of your project?',
     },
+    {
+      type: 'autocomplete',
+      name: 'packageManager',
+      message: 'Which package manager would you like to use?',
+      choices: PACKAGE_MANAGERS.map((pm) => ({ title: pm, value: pm })),
+      suggest: (input, choices) => Promise.resolve(choices.filter((c) => c.title.includes(input))),
```

Comment:
How would this work for Deno?

I noticed this line in the [ticket](https://github.com/thisdot/starter.dev/issues/1271)

For Deno or other projects, I should be presented with appropriate options for that kit.

Is there a way to conditionally show package manger options based on the kit chosen?